src/styles/globals.css

```diff
@@ -10,3 +10,8 @@
   --starick-white: rgb(243, 244, 246);
   --starick-black: rgb(3, 7, 18);
 }
+
+.my-link {
+  color: orange;
+  text-decoration: underline;
+}
```

Comment:
You might have to wrap this in a `@layer components {}`

https://tailwindcss.com/docs/adding-custom-styles#adding-component-classes